daffodil-core/src/main/scala/org/apache/daffodil/dsom/SequenceGroup.scala

```diff
@@ -245,6 +248,31 @@ abstract class SequenceTermBase(
       maybeCheckBitOrderAndCharset)
   }
 
+  private val layeredSequenceAllowedProps = Set("ref", "layerTransform", "layerEncoding", "layerLengthKind", "layerLength", "layerLengthUnits", "layerTerminator")
+
+  final lazy val maybeLayerTransformerEv: Maybe[LayerTransformerEv] = {
+    if (maybeLayerTransformEv.isEmpty) Maybe.Nope
```

Comment:
You can't SDE because they could be in scope for use by layers elsewhere in the schema. 
If they are expressed directly on this specific layered sequence, then we can SDW about them.